packages/modules/devices/fems/config.py

```diff
@@ -11,12 +11,16 @@ def __init__(self, ip_address: Optional[str] = None, password: Optional[str] = N
 
 class Fems:
     def __init__(self,
-                 name: str = "openEMS, Fenecon FEMS, CENTURIO 10, Kaco Hy-Control",
+                 name: str = "Fenecon/FEMS",
```

Comment:
```suggestion
                 name: str = "openEMS, FEMS, CENTURIO 10, Kaco Hy-Control",
```
Ein Kunde weiß wahrscheinlich nicht, dass sein Kaco Wechselrichter die gleiche Schnittstelle wie FEMS hat und wird nicht darunter suchen.

Reply:
> Ein Kunde weiß wahrscheinlich nicht, dass sein Kaco Wechselrichter die gleiche Schnittstelle wie FEMS hat und wird nicht darunter suchen.

Ok, d.h. damit der Herstellername sauber unter Hersteller im Frontend angezeigt wird, sollte da dann openEMS/FEMS/Centurio/Kaco hin?

Reply:
Ja, bitte.

Reply:
> Ja, bitte.

Erledigt.